src/components/RichTextPopup/RichTextPopup.js

```diff
@@ -40,27 +43,43 @@ const RichTextPopup = () => {
     };
 
     core.addEventListener('editorSelectionChanged', handleSelectionChange);
-    return () => core.removeEventListener('editorSelectionChanged', handleSelectionChange);
+    return () =>
+      core.removeEventListener('editorSelectionChanged', handleSelectionChange);
   }, []);
 
   useEffect(() => {
     const handleTextChange = () => {
       if (annotationRef.current?.isAutoSized() && popupRef.current) {

```

Comment:
I think `spaceAdjustment` is always `0` since `symbolsVisible` is always `false` because `symbolsVisible` is missing in the dependency array.
However, When I was testing I didn't see anything weird with `spaceAdjustment` always being `0`. So I'm wondering if we actually need the adjustment?

Reply:
`spaceAdjustment` has been removed.  See my next commnet